specs/core/0_beacon-chain.md

```diff
@@ -198,32 +198,30 @@ The following values are (non-configurable) constants used throughout the specif
 | - | - |
 | `GENESIS_SLOT` | `Slot(0)` |
 | `GENESIS_EPOCH` | `Epoch(0)` |
-| `BLS_WITHDRAWAL_PREFIX` | `0` |
+| `BLS_WITHDRAWAL_PREFIX` | `Bytes1()` |
```

Comment:
Like `GENESIS_SLOT`, I think should be `Bytes1(0)` to show that it is a selected value. Depending on the network, you could definitely use different withdrwaal prefixes as versions.